src/utils/utils.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as cp from 'child_process';
+
+export async function executeCommand(command: string, cwd?: string): Promise<string> {
+	return new Promise<string>((resolve, reject) => {
+		cp.exec(command, { maxBuffer: 500 * 1024, cwd: cwd }, (error: Error, stdout: string, stderr: string) => {
+			if (error) {
+				reject(error);
+				return;
+			}
+			if (stderr && stderr.length > 0) {
+				reject(new Error(stderr));
+				return;
+			}
+			resolve(stdout);
+		});
+	});
+}
+
+/**
+ * Generates a quoted full name for the object
+ * @param schema of the object
+ * @param objectName object chosen by the user
+ * @returns the quoted and escaped full name of the specified schema and object
+ */
+export function generateQuotedFullName(schema: string, objectName: string): string {
+	schema = schema.replace(/\]/g, ']]');
```

Comment:
Utils already has a function to do the escaping (escapeClosingBrackets) so I'd just use that

```
return `[${escapeClosingBrackets(schema)}].[${escapeClosingBrackets(objectName)}]`;
```